### ISSUE TITLE
Pancreatic α-cell hyperplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7020]
**Preferred term label:**
Pancreatic alpha-cell hyperplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
A diffuse and specific increase in the number of alpha-cells.
Comment: Alpha cells are glucagon-secreting endocrine cells in the pancreatic islets of the pancreas. 
PMID: 24285676

**Parent term (use hpo.jax.org/app)**
Abnormal pancreas morphology HP:0012090

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619290 , 3/3 PMID:25695890 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
